frameworks/Java/javalin/src/main/java/com/lostsys/test/javalin/Bench.java

```diff
@@ -0,0 +1,30 @@
+package com.lostsys.test.javalin;
+
+import io.javalin.Javalin;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Bench {
+
+    public static void main(String[] args) {
+
+        Javalin app = Javalin.create().start(8080);
+        app.get("/plaintext", ctx -> ctx.result("Hello, World!"));
+
+        app.get("/json", ctx -> {
+            Map<String, String> hello = new HashMap<>();
+            hello.put("message", "Hello, World!");
```

Comment:
Use `Collections.singletonMap("message", "Hello, World!")` instead (from `java.util.Collections`).  `singletonMap` uses a dedicated `Map` implementation for a single key and value that is much more efficient than `HashMap`.  The impact of this choice on the JSON test results is measurable in some of the faster Java frameworks, IIRC.

If this were a more recent version of Java (after 8), I would say to use `Map.of("message", "Hello, World!")`.

Reply:
Done. Thanks for the tip!